File: import/src/main/java/de/helfenkannjeder/helfomat/domain/Group.java
package de.helfenkannjeder.helfomat.domain;

/**
 * @author Valentin Zickner
 */
public class Group {
    private String name;
    private String description;

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getDescription() {
        return description;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) {
            return true;
        }
        if (!(o instanceof Group)) {
            return false;
        }

        Group group = (Group) o;

        if (!name.equals(group.name)) {
            return false;
        }
        return description != null ? description.equals(group.description) : group.description == null;
    }

    @Override
    public int hashCode() {
        int result = name.hashCode();
        result = 31 * result + (description != null ? description.hashCode() : 0);
        return result;
    }
}

File: helfomat-core/src/main/java/de/helfenkannjeder/helfomat/core/organization/Group.kt
package de.helfenkannjeder.helfomat.core.organization

/**
 * @author Valentin Zickner
 */
data class Group(
    var name: String,
    var description: String?,
    var contactPersons: List<ContactPerson> = emptyList(),
    var minimumAge: Int? = null,
    var maximumAge: Int? = null,
    var website: String? = null
)